packages/core/src/framebuffer/FramebufferSystem.js

```diff
@@ -169,6 +169,44 @@ export class FramebufferSystem extends System
         }
     }
 
+    /**
+     * Copies the pixels in the source color buffer to the destination
+     * color buffer. This creates two temporary framebuffers to complete
+     * the operation.
+     *
+     * @param {PIXI.ColorBuffer} srcBuffer
+     * @param {PIXI.ColorBuffer} dstBuffer
+     */
+    blitColorBuffer(srcBuffer, dstBuffer)
+    {
+        const srcFb = new Framebuffer().addColorBuffer(srcBuffer);
```

Comment:
Better way is to use static methods for creating new instances with options

`const srcFb = Framebuffer.fromColorBuffer(srcBuffer);`